contracts/echidna/Hevm.sol

```diff
@@ -0,0 +1,22 @@
+// @unsupported: ovm
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.7.6;
+
+interface Hevm {
```

Comment:
what does Hevm stand for?

Reply:
Haskell EVM. Echidna uses it as an execution environment under the hood. 

It has some special "cheat codes" that we use in the fuzzing tests to advance time, etc:
https://github.com/dapphub/dapptools/blob/master/src/hevm/README.md#cheat-codes